doc/docs/master/reference/pipeline_spec.md

```diff
@@ -399,13 +402,15 @@ is marked as failed.
 
 ### Job Timeout (optional)
 
-`job_timeout` is a string (e.g. `1s`, `5m`, or `15h`) that determines the
-maximum execution time allowed for a job. It differs from `datum_timeout`
-in that the limit gets applied across all workers and all datums. That
-means that you'll need to keep in mind the parallelism, total number of
-datums, and execution time per datum when setting this value. Keep in
-mind that the number of datums may change over jobs. Some new commits may
-have a bunch of new files (and so new datums). Some may have fewer.
+`job_timeout` determines the maximum execution time allowed for a job. It
```

Comment:
Can you double check with eng on this. Is job_timeout "wall time",  meaning that if you set it for 1 hour, the jobs will get killed if it is not complete in 1hr -- as opposed to cumulative processing time across parallel workers. The latter wouldn't really make sense because it'd be very unlike to be useful to users. 

If wall time is correct, then:

`applied across all workers and all datums. Therefore, when you set this value, you need to
consider the parallelism, total number of datums, and execution time per datum.` can be clarified a bit.

Reply:
Confirming that this is wall time.